docs/src/reference/export.md

```diff
@@ -24,10 +24,16 @@ evaluate_at_grid_nodes
 ```
 
 ## VTK Export
-
 ```@docs
-vtk_grid(filename::AbstractString, grid::Grid{dim,C,T}; compress::Bool) where {dim,C,T} 
-vtk_point_data
-vtk_cellset
-vtk_cell_data_colors
+VTKFile
+VTKFileCollection
+addstep!
+write_solution
+write_projection
+write_celldata
+write_nodedata
+Ferrite.write_cellset
+Ferrite.write_nodeset
+Ferrite.write_dirichlet
+Ferrite.write_cell_colors
```

Comment:
There is `celldata` (no `_`) and `cell_colors`. Perhaps be consistent? I think I prefer with the `_` but not sure. `cellset` and `nodeset` matches `getcellset` etc so perhaps they should be without..